gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
> six is already required by the base package (there's a comment "raise_from on Python 3" in setup.py) as well as numerous other libraries required by gmusicapi so it won't actually add another compat library. :)

I didn't say add, I said use. Quite different. The reason ``six.raise_from()`` is used is due to a bug/quirk in python-future. That was used because it was a non-trivial thing, unlike getting the Python version. If anything, the more "Zen" way would be to try/except ImportError, but I prefer to have it be obvious to even novices what is going on and why if read.

> six is already required by the base package (there's a comment "raise_from on Python 3" in setup.py) as well as numerous other libraries required by gmusicapi so it won't actually add another compat library. :)

I said use, not add. ``six.raise_from()`` had to be used because of a bug/quirk in future's ``raise_from`` (a couple open issues on their issue tracker if interested). The alternative would have been to basically reimplement it in a manual compat layer, which would be pointless : )

A good way to think of it is that ``future`` is the primary compat layer. If something can't be solved by future, try ``six`` which is already there. If neither of those work, roll our own. In this case, ``future`` has what was needed and in a way that is probably clearer.